railties/lib/rails/generators/rails/app/templates/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  db:
+    # Please change the image you want
+    image: postgres
+  web:
+    build: .
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
```

Comment:
The Rails server listens on port 3000 by default. We needn’t pass the port option to `rails s` here.